### PR TITLE
[WIP] Updated to include the coding school name

### PR DESCRIPTION
**Pull request by bas**
Updated the readme to add Tjejer Kodar

### CODE REVIEW

**Comment by CissiWong** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Welcome to my GitHub Pages site
 
-Hello World!
+Hello Tjejer Kodar!
```
Comment (bas) [inline README.md]: @bas wow! you really know some Swedish. Never expected that from a Dutch person. 👍 🚀 🎉

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 You can use the [editor on GitHub](https://github.com/bas/bas.github.io/edit/master/README.md) to maintain and preview the content for your website in Markdown files.
```
Comment (bas) [inline README.md]: @bas not sure if this is the right link.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Welcome to my GitHub Pages site
 
-Hello World!
+Hello Tjejer Kodar!
```
Comment (CissiWong) [inline README.md]: @bas Impressive!

File: README.md (hunk 1 above)
Comment (CissiWong) [inline README.md]: @bas  Are you sure about this spelling though?

Comment (bas) [reply README.md]: @CissiWong Shall I change Hello to Hey or is this ok?

**Pull request merged**